### ISSUE TITLE
Allow font size to be overridden with browser settings

### ISSUE THREAD

**Issue by phudson-he** [archesproject/arches#9785]
By using font sizes in css/scss with "px" means that a user cannot override font sizes using browser accessibility settings.

I've done a bit of research on this and "rem" should be used instead of "px". If an item has specific scaling, depending on its parent size, then "em" should be used. From experience, I would suggest that "rem" would be mostly used.

By changing all font sizes to rem values, allows a user to override (eg. increase font sizes to aid visually). This font size setting is available in all the major web browsers.

**Comment by johnatawnclementawn** [comment]
Resolved in #9862
